### PR TITLE
[DONT DELETE][WIP] web3.js 2.0.0-alpha.1

### PR DESCRIPTION
**Pull request by kvhnuke**
Too many things broken on web3 side
Broken
- [x] Metamask support
- [x] ENS

Working
- [x] http-provider
- [x] ws-provider
- [x] all custom functions
### Devop

- [ ] Updated CHANGELOG.md
- [ ] Add PR label

### CODE REVIEW

File: src/layouts/AccessWalletLayout/components/MetamaskModal/MetamaskModal.vue [hunk 1]
```diff
@@ -149,7 +149,8 @@ export default {
       }
     },
     signIn(web3) {
-      new Web3(web3.currentProvider).eth
+      console.log(Web3);
```
Comment (gamalielhere) [inline src/layouts/AccessWalletLayout/components/MetamaskModal/MetamaskModal.vue]: Console. 😉

File: src/layouts/AccessWalletLayout/components/PasswordModal/PasswordModal.vue [hunk 2]
```diff
@@ -102,6 +102,7 @@ export default {
         worker.onerror = function(e) {
           e.preventDefault();
           self.spinner = false;
+          console.log(e);
```
Comment (gamalielhere) [inline src/layouts/AccessWalletLayout/components/PasswordModal/PasswordModal.vue]: Another one

File: src/main.js [hunk 3]
```diff
@@ -105,23 +105,23 @@ const vue = new Vue({
...
-      walletType: store.getters.account.identifier
-    };
-    return new Promise(resolve => {
-      vue.$eventHub.$emit('issueModal', event, resolve);
-    }).then(res => {
-      return res === true ? event : null;
-    });
-  }
-});
+// Sentry.init({
```
Comment (gamalielhere) [inline src/main.js]: Should this be uncommented?

**Comment by kvhnuke** [comment]
waiting on https://github.com/ethereum/web3.js/issues/2661

**Comment by nivida** [comment]
@kvhnuke Is this the last thing to fix? Would be great if MEW would be a part of the discussion here: https://github.com/ethereum/web3.js/issues/2684

**Comment by nivida** [comment]
Btw.: I've seen you are using the package ``ethereum-ens``. Web3.js would also provide an ENS module.

**Comment by kvhnuke** [comment]
@nivida sorry just saw your messages, I've been merging things in as you release new web3 versions and finding bugs as I go, didnt really have a chance to go through everything. Will definitely look into https://github.com/ethereum/web3.js/issues/2684 and `web3.js ens`

**Pull request closed without merging**